### PR TITLE
Updated for godocs..

### PR DESCRIPTION
**Pull request by garimakhulbe**
Added comments for godocs.

### CODE REVIEW

**Comment by ahmetalpbalkan** [comment]
only one file per package should have the godoc comment, not all files in the package. actually the convention is to have a `doc.go` in each package that just has the godoc comment, but no need to do it here as long as it's on a single file.

also pls join the azure linux team at aka.ms/azuregithub.

File: arm/network/client.go [hunk 1]
```diff
@@ -1,3 +1,10 @@
+// Package network implements the Azure ARM Network service API version
+// 2015-06-15.
+//
+// The Microsoft Azure Network management API provides a RESTful set of web
+// services that interact with Microsoft Azure Networks service to manage
+// your network resrources. The API has entities that capture the
+// relationship between an end user and the Microsoft Azure Networks service..
```
Comment (ahmetalpbalkan) [inline arm/network/client.go]: seems like we have a duplicate `..` problem in some packages at the end.

File: arm/compute/client.go [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// Package compute implements the Azure ARM Compute service API version
+// 2015-06-15.
+//
+// The Compute Management Client..
```
Comment (ahmetalpbalkan) [inline arm/compute/client.go]: this doesn't look very good. :( maybe omit what services describe in swagger and just keep the first line you have? It just seems sufficient to me.

Comment (ahmetalpbalkan) [inline arm/compute/client.go]: it's actually not a big deal, we can merge as the first line is what's going to be shown at https://godoc.org/github.com/Azure/azure-sdk-for-go/arm

Comment (brendandixon) [inline arm/compute/client.go]: Let's merge. The Swagger documentation needs work.

File: arm/network/client.go (hunk 1 above)
Comment (garimakhulbe) [inline arm/network/client.go]: I fixed this in latest PR.

**Pull request merged**